docs/core/tools/dotnet-new.md

```diff
@@ -53,6 +60,36 @@ The command calls the [template engine](https://github.com/dotnet/templating) to
 
 The template to instantiate when the command is invoked. Each template might have specific options you can pass. For more information, see [Template options](#template-options).
 
+If the `TEMPLATE` value isn't an exact match on text in the **Templates** or **Short Name** column, a substring match is performed on those two columns.
+
+# [.NET Core 2.2](#tab/netcore22)
+
+The command contains a default list of templates. Use `dotnet new -l` to obtain a list of the available templates. The following table shows the templates that come pre-installed with the .NET Core SDK 2.2.100. The default language for the template is shown inside the brackets.
+
+|Template description                          | Template name     | Languages    |
+|----------------------------------------------|-------------------|--------------|
+| Console Application                          | `console`         | [C#], F#, VB |
+| Class library                                | `classlib`        | [C#], F#, VB |
+| Unit Test Project                            | `mstest`          | [C#], F#, VB |
+| NUnit 3 Test Project                         | `nunit`           | [C#], F#, VB |
+| NUnit 3 Test Item                            | `nunit-test`      | [C#], F#, VB |
+| xUnit Test Project                           | `xunit`           | [C#], F#, VB |
+| Razor Page                                   | `page`            | [C#]         |
+| MVC ViewImports                              | `viewimports`     | [C#]         |
+| MVC ViewStart                                | `viewstart`       | [C#]         |
+| ASP.NET Core Empty                           | `web`             | [C#], F#     |
+| ASP.NET Core Web App (Model-View-Controller) | `mvc`             | [C#], F#     | 
+| ASP.NET Core Web App                         | `webapp`, `razor` | [C#]         |
+| ASP.NET Core with Angular                    | `angular`         | [C#]         |
+| ASP.NET Core with React.js                   | `react`           | [C#]         |
+| ASP.NET Core with React.js and Redux         | `reactredux`      | [C#]         |
+| Razor Class Library                          | `razorclasslib`   | [C#]         |
+| ASP.NET Core Web API                         | `webapi`          | [C#], F#     |
+| global.json file                             | `globaljson`      |              |
+| NuGet Config                                 | `nugetconfig`     |              |
+| Web Config                                   | `webconfig`       |              |
+| Solution File                                | `sln`             |              |
+
```

Comment:
Should this table be the same as what is output by `dotnet new -l` ?

Reply:
In its current state, it doesn't match. @mairaw What's the intent?

Reply:
yes, it should match. Good catch @seancpeters. The intent is to list the available templates.